src/main/java/seedu/address/storage/JsonSerializableRoomList.java

```diff
@@ -42,7 +42,7 @@ public JsonSerializableRoomList(RoomList source) {
      */
     public RoomList toModelType() throws IllegalValueException {
         RoomList roomList = new RoomList();
-        roomList.getRoomDisplayList().clear();
+        //roomList.getRoomDisplayList().clear();
```

Comment:
Maybe it would be good to remove codes that we commented out? Otherwise, we may completely forget about it.